.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
     "parser": "babel-eslint",
+    "extends": [
+      "defaults/configurations/google"
```

Comment:
Hmm my previous impression is we use gjslint as well, so we do follow google's lint configs. I don't have strong opinion but I tend to pick a default and tweak a bit for our own needs.